### DISCUSSION TITLE
Aide

### DISCUSSION THREAD

**Discussion by Oknessman** [pkgxdev/pkgx#955] [Help]
Nous sommes vraiment chanceux de faire parti early du voyage avec tea

**Comment by hanriiver** [comment]
Please tell me this on Twitter or Discord.
It is not issue

**Comment by Oknessman** [comment]
Ok, I will.

Le mar. 30 janv. 2024 à 10:31, hanriiver ***@***.***> a
écrit :

**Comment by tandotendo** [comment]
Let's continue this on Discord, if you want you can chat with me there. Thanks

**Comment by aleshah365** [comment]
Most of people active on discord
